### ISSUE TITLE
SFTPGO_LOG_LEVEL is not used

### ISSUE THREAD

**Issue by jsalgado78** [drakkan/sftpgo#1004]
SFTPGO_LOG_LEVEL environment variable isn't used but it's documented:

`
--log-level string. Set the log level. Supported values: debug, info, warn, error. Default debug or the value of SFTPGO_LOG_LEVEL environment variable.`

For example:
```
$ docker run -it --rm -e SFTPGO_LOG_LEVEL=error -e SFTPGO_LOG_VERBOSE=0 drakkan/sftpgo:v2.3.5-alpine-slim 
{"level":"info","time":"2022-09-29T11:43:47.825","sender":"httpd","message":"using memory reset code manager"}
{"level":"info","time":"2022-09-29T11:43:47.825","sender":"httpd","message":"using memory OIDC manager"}
{"level":"info","time":"2022-09-29T11:43:47.883","sender":"httpd","message":"server listener registered, address: [::]:8080 TLS enabled: false"}

$ docker run -it --rm -e SFTPGO_LOG_LEVEL=error drakkan/sftpgo:v2.3.5-alpine-slim 
{"level":"info","time":"2022-09-29T11:45:44.761","sender":"httpd","message":"using memory OIDC manager"}
{"level":"info","time":"2022-09-29T11:45:44.762","sender":"sftpd","message":"No host keys configured and \"id_rsa\" does not exist; try to create a new host key"}
{"level":"debug","time":"2022-09-29T11:45:44.762","sender":"util","message":"found share data path for name \"static\": \"/usr/share/sftpgo/static\""}
{"level":"debug","time":"2022-09-29T11:45:44.762","sender":"util","message":"found share data path for name \"templates\": \"/usr/share/sftpgo/templates\""}
{"level":"debug","time":"2022-09-29T11:45:44.762","sender":"util","message":"found share data path for name \"openapi\": \"/usr/share/sftpgo/openapi\""}

```

**Comment by drakkan** [comment]
Try an edge variant. This is not included in the 2.3 branch

**Comment by jsalgado78** [comment]
It works with edge images
Thanks
